libsolidity/interface/CompilerStack.cpp

```diff
@@ -215,6 +216,8 @@ bool CompilerStack::analyze()
 				{
 					contract->setDevDocumentation(Natspec::devDocumentation(*contract));
 					contract->setUserDocumentation(Natspec::userDocumentation(*contract));
```

Comment:
This is not strictly related to this PR, but why is the documentation stuff in this conditional?